config/.env.example

```diff
@@ -0,0 +1,2 @@
+KEYSTORE_PASSWORD=default-keystore-password
+DB_PASSWORD=default-db-password
```

Comment:
This is awesome! Would you mind adding instructions to the README?

Reply:
yea good call